### PR TITLE
auth checking inputs

### PR DESCRIPTION
**Pull request by Nikita-Koval**

### CODE REVIEW

File: src/App.scss [hunk 1]
```diff
@@ -44,4 +46,14 @@ body {
   max-height: 400px;
   width: 45%;
   margin-top: 8rem;
+}
+
```
Comment (usatovaDasha) [inline src/App.scss:51]: remove empty lines

File: src/App.scss [hunk 2]
```diff
@@ -44,4 +46,14 @@ body {
...
+
+
+.extBtn {
+  margin-top: 1em;
```
Comment (usatovaDasha) [inline src/App.scss:53]: add empty line

File: src/App.scss [hunk 3]
```diff
@@ -44,4 +46,14 @@ body {
...
+  .MuiButton-contained {
+    max-height: 50px;
+    width: 120px;
+}
```
Comment (usatovaDasha) [inline src/App.scss]: check tabs

File: src/components/allNotes/AllNotes.js [hunk 4]
```diff
@@ -0,0 +1,24 @@
+import React, { useState } from 'react';
+import { Link, useHistory } from "react-router-dom";
+import Button from '@material-ui/core/Button';
+import Header from '../header/Header';
+import FormList from '../allNotesFormList/FormList';
+import axios from 'axios';
+
+const AllNotes = () => {
+const history = useHistory();
```
Comment (usatovaDasha) [inline src/components/allNotes/AllNotes.js]: check tabs

File: src/components/allNotes/AllNotes.js [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+
+return (
+  <div>
+    <Header>
+      Приёмы
+      <span className='extBtn'>
```
Comment (usatovaDasha) [inline src/components/allNotes/AllNotes.js]: why need span?

File: src/components/allNotesFormList/FormList.js [hunk 6]
```diff
@@ -0,0 +1,21 @@
+import React, { useState } from 'react';
+import { 
+  BrowserRouter, 
+  Switch, 
+  Route, 
+  Redirect } from "react-router-dom";
```
Comment (usatovaDasha) [inline src/components/allNotesFormList/FormList.js]: } from "react-router-dom";
on new line

File: src/components/allNotesFormList/FormList.js [hunk 7]
```diff
@@ -0,0 +1,21 @@
...
+
+const FormList = () => {
```
Comment (usatovaDasha) [inline src/components/allNotesFormList/FormList.js:9]: check tabs

File: src/components/login/Login.js [hunk 8]
```diff
@@ -12,28 +12,32 @@ const history = useHistory();
...
-        localStorage.setItem('token', res.token.token)
-        history.push("/allNotes")
-      });
+  const handleSubmit = (e) => {
+    if((!/[A-Z]/.test(password)) 
+      && (!/[a-z]/.test(password)) 
+      || (!/[0-9]/.test(password)) 
+      || ((login.length && password.length) < 6)
+    ) {
+        alert('Некорректные данные. Пароль должен содержать латинские символы и цифры.') } else {
```
Comment (usatovaDasha) [inline src/components/login/Login.js]: } else {
on new line

File: src/components/allNotes/AllNotes.js (hunk 5 above)
Comment (Nikita-Koval) [reply src/components/allNotes/AllNotes.js]: For margin material-ui element

**Pull request merged**